src/amplitude/timeline.py

```diff
@@ -43,7 +43,7 @@ def remove(self, plugin):
     def flush(self):
         for destination in self.plugins[PluginType.DESTINATION]:
             try:
-                destination.flush()
+                return destination.flush()
```

Comment:
Since timeline could have multiple destination plugins, return here may end the function earlier and only return future of flush call. Should return a list of future for all destination plugins